poly-synth/poly-synth.html

```diff
@@ -1,42 +0,0 @@
-<!--
```

Comment:
It got renamed to index.html, so you could automatically navigate to the page, but it was left out of this push by mistake. Sorry!